sleap/gui/widgets/video.py

```diff
@@ -970,8 +972,16 @@ def selectInstance(self, select: Union[Instance, int]):
         Returns:
             None
         """
+        
         for idx, instance in enumerate(self.all_instances):
+            if isinstance(select, int):
+                instance.selected = select == idx
+            elif isinstance(select, Instance):
+                instance.selected = select == instance.instance
+            elif isinstance(select, InstanceGroup):
+                instance.selected = True if instance in select else False
```

Comment:
`InstanceGroup` stores `Instance` objects not `QtInstance` objects, so we need to use the `QtInstance.instance` attribute to get the associated `Instance`. 

Also, when using the `in` keyword in Python, the dunder method `__contains__` is called on the object that is being checked for containment. If `__contains__` is not implemented, Python falls back to using the `__iter__` method to iterate over the elements (if available) and checks each element. If `__iter__` is also not implemented, Python then tries to use the `__getitem__` method to check for containment by iterating over index values starting from zero.

Neither `InstanceGroup.__contains__` not `InstanceGroup.__iter__` is implmented and `InstanceGroup.__getitem__` unfortunately only handles this in the case of the `Instance` selected being the first instance in `InstanceGroup.instances` (we iterate to too large a number otherwise and get a `KeyError`). Let's just use `in InstanceGroup.instances` for now.
```suggestion
                instance.selected = True if instance.instance in select.instances else False
```